cluster/cluster.yaml

```diff
@@ -3,6 +3,7 @@ Description: Kubernetes cluster
 Metadata:
   Tags:
     InfrastructureComponent: "true"
+    application: "kubernetes"
```

Comment:
We have this defined individually below for some specific resources. Would make sense to remove those per resource tags where it would be overwritten by this. For instance we have some `application: kube-apiserver` tags defined which are no longer correct and should be dropped.

Reply:
Agreed. Will remove per-resource tags for this template and will keep only the "top level" tag already added.